test/helpers/fixture_set.rb

```diff
@@ -0,0 +1,76 @@
+require 'helpers/kubeclient_helper'
+
+module FixtureSetAssertions
+  class FixtureSet
+    class FixtureSetError < StandardError; end
+    include KubeclientHelper
+    include Minitest::Assertions
+    attr_writer :assertions
+
+    def initialize
+      raise NotImplementedError
+    end
+
+    def assertions
+      @assertions ||= 0
+    end
+
+    def namespace
+      raise FixtureSetError.new("@namespace must be set in initializer") if @namespace.blank?
+      @namespace
+    end
+
+    def app_name
+      raise FixtureSetError.new("@app_name must be set in initializer") if @app_name.blank?
+      @app_name
+    end
+
+    def refute_resource_exists(type, name, beta: false)
+      client = beta ? v1beta1_kubeclient : kubeclient
```

Comment:
Arrghhh, this is so ugly due to upstream 😔  Can we auto-detect this somehow instead of passing `beta: true`? I'm afraid of that staying around forever..